src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -1550,29 +1557,187 @@ public void testInvalidArrayInitWithTrueStrictCondition1()
+
+    @Test
+    public void testInvalidArrayInitWithFalseStrictCondition()
+            throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(IndentationCheck.class);
+
+        checkConfig.addProperty("arrayInitIndent", "4");
+        checkConfig.addProperty("basicOffset", "4");
+        checkConfig.addProperty("braceAdjustment", "0");
+        checkConfig.addProperty("caseIndent", "4");
+        checkConfig.addProperty("forceStrictCondition", "false");
+        checkConfig.addProperty("lineWrappingIndentation", "4");
+        checkConfig.addProperty("tabWidth", "4");
+        checkConfig.addProperty("throwsIndent", "4");
+        final String fileName = getPath("InputIndentationInvalidArrayInitIndent.java");
+        final String[] expected = {
+            "21:3: " + getCheckMessage(MSG_ERROR, "member def type", 2, 4),
+            "22:7: " + getCheckMessage(MSG_ERROR, "member def type", 6, 4),
+            "24:3: " + getCheckMessage(MSG_ERROR, "member def type", 2, 4),
+            "28:7: " + getCheckMessage(MSG_ERROR, "member def type", 6, 4),
+            "29:9: " + getCheckMessage(MSG_CHILD_ERROR_MULTI, "array initialization", 8,
+                "10, 34, 36"),
+            "30:5: " + getCheckMessage(MSG_ERROR_MULTI, "array initialization rcurly", 4, "6, 10"),
+            "33:10: " + getCheckMessage(MSG_CHILD_ERROR_MULTI, "array initialization", 9,
+                "8, 31, 33"),
+            "34:8: " + getCheckMessage(MSG_CHILD_ERROR_MULTI, "array initialization", 7,
+                "8, 31, 33"),
+            "35:10: " + getCheckMessage(MSG_CHILD_ERROR_MULTI, "array initialization", 9,
+                "8, 31, 33"),
+            "40:3: " + getCheckMessage(MSG_ERROR_MULTI, "array initialization lcurly", 2, "4, 8"),
+            "44:7: " + getCheckMessage(MSG_ERROR_MULTI, "array initialization rcurly", 6, "4, 8"),
+            "48:3: " + getCheckMessage(MSG_ERROR_MULTI, "array initialization lcurly", 2, "4, 8"),
+            "52:21: " + getCheckMessage(MSG_CHILD_ERROR_MULTI, "array initialization", 20,
+                "8, 31, 33"),
+            "53:5: " + getCheckMessage(MSG_CHILD_ERROR_MULTI, "array initialization",
+                4, "8, 31, 33"),
+            "58:7: " + getCheckMessage(MSG_CHILD_ERROR_MULTI, "array initialization",
+                6, "8, 31, 33"),
+            "63:3: " + getCheckMessage(MSG_ERROR, "member def type", 2, 4),
+            "65:7: " + getCheckMessage(MSG_ERROR, "member def type", 6, 4),
+            "66:3: " + getCheckMessage(MSG_ERROR_MULTI, "array initialization rcurly", 2, "6, 10"),
+            "69:7: " + getCheckMessage(MSG_CHILD_ERROR_MULTI, "array initialization",
+                6, "8, 36, 38"),
+            "76:11: " + getCheckMessage(MSG_CHILD_ERROR_MULTI, "array initialization",
+                10, "12, 24, 26"),
+        };
+
+        // Test input for this test case is not checked due to issue #693.
+        verifyWarns(checkConfig, fileName, expected);
```

Comment:
I am confused on comment, can we use verifyWarns or not?
I thought this issue blocks usage of verifyWarns and we should use `verify` method.

Reply:
please remove this comment, it is ok that test does not cover all.
Main point here is that it is using `verifyWarns` that is used by most of tests, so all is good.

just remove this comment. same for another test method.